### PR TITLE
Add "About" button into tray menu

### PR DESCRIPTION
**Pull request by BlueManCZ**
To be honest, this is the very first code I wrote in the go language, and I'm nicely surprised, how straightforward and intuitive it is.

I'm not sure about that `VERSION` constant in `constants.go`. Should I try to parse it from PKGBUILD? What do you think?

### CODE REVIEW

**Comment by BlueManCZ** [comment]
I just noticed it works only once. When you click to the button again, nothing happens. Strange, I will try to fix it.

**Comment by Merrit** [comment]
Hard coding the version feels like it would be error prone. Maybe it would be better to open to the [releases page](https://github.com/Merrit/nyrna/releases) which has all the changelogs? Benefit of not having to worry about the version being coded into every release..

Also this library supports error handling, so it should probably be something like:

``` go
err := open.Run("https://github.com/Merrit/nyrna/releases")
if err != nil {
    log.Println("Error opening Nyrna changelog: ", err)
    // maybe also use Notify() to send a user-facing message
    // about the failure? Not sure what kind of error it returns..
}
```

**Comment by BlueManCZ** [comment]
Alright, I just wanted to let users know, what version of the app they are using. You usually find this information in "about" section in other apps so I thought I would be nice to have it here as well. There could be also a notification of the new version based on this variable.

Snímek z 2020-07-27 16-17-27
Snímek z 2020-07-27 16-17-00

**Comment by BlueManCZ** [comment]
Maybe it would be possible to somehow automate this version variable in the build process.
Something like `go build --version 1.2`, if you know, what I mean? But I'm not very familiar with go yet.

**Comment by Merrit** [comment]
You've convinced me, you are right :)

Could you add the error handling, since right now it keeps printing "nil"?

**Comment by BlueManCZ** [comment]
Sure thing.

**Comment by BlueManCZ** [comment]
Should we keep `v` letter in the VERSION constant

`const VERSION string = "v1.2"`

or it would be better without it and add it in URL?

`open.Start(HOMEPAGE + "/releases/tag/v" + VERSION)`

**Comment by Merrit** [comment]
Looks great, thank you! :)

File: systray.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 func onReady() {
 	systray.SetIcon(icon.Data)
 	mRebind := systray.AddMenuItem("Change Hotkey", "Choose a new hotkey")
-	mAbout := systray.AddMenuItem("About Nyrna " + VERSION, "Open changelog")
+	mAbout := systray.AddMenuItem("About Nyrna "+VERSION, "Open changelog")
```
Comment (BlueManCZ) [inline systray.go:20]: This doesn't look right. :smiley: Spaces around operators should be the same across the entire code, don't you think? You could as well do `mAbout:=systray...`

Comment (Merrit) [reply systray.go:20]: The linter was collapsing this item because it was about + VERSION was considered 1 argument. Should be fixed now.

Nice catch :+1:

**Pull request merged**